src/hooks/useSubjectProjects.tsx

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from 'react-query';
+import { useActiveAccount } from './useActiveAccount';
+import { useMe } from './useMe';
+import { useHttpClient } from './useHttpClient';
+
+export interface Project {
+  id: string;
+  name: string;
+}
+
+interface ProjectsResponse {
+  items: Project[];
+}
+
+export function useSubjectProjects() {
+  const { account, accountHeaders } = useActiveAccount();
+  const { data: subjects } = useMe();
+  const { httpClient } = useHttpClient();
+
+  return useQuery(
+    `${account?.id}-projects`,
+    () =>
+      httpClient
+        .get<ProjectsResponse>(
+          `/v1/projects?id=${subjects?.map((s) => s.projectId).join(',')}`,
+          { headers: accountHeaders },
+        )
+        .then((res) => res.data.items),
+    {
+      enabled: !!accountHeaders && !!subjects?.[0]?.projectId,
```

Comment:
Should this be: 

```suggestion
      enabled: !!accountHeaders && !!subjects?.some(subject => subject.projectId),
```

Reply:
Yea good catch.  Well, since we're sending all `projectId`s in the URL, I suppose it should even be `.every(...)`.  In my mind, a record in here without `projectId` would "never" happen.  But this change is much better defensive wise - will fix up in upcoming PR